src/github.com/getlantern/peerscanner/host.go

```diff
@@ -430,3 +434,13 @@ func (h *host) doIsAbleToProxy() (bool, bool, error) {
 
 	return true, false, nil
 }
+
+// fallbackCountry returns the country code of a fallback if it follows the
+// usual naming convention.
+func fallbackCountry(name string) string {
```

Comment:
I just moved this because I noticed that this is the file where it was being used, not the file where it lived before.